apis/r/README.md

```diff
@@ -14,7 +14,7 @@ install.packages('tiledbsoma', repos = c('https://tiledb-inc.r-universe.dev',
                                          'https://cloud.r-project.org'))
 ```
 
-Installing from source on Unix-like platforms requires `cmake` and `git`.
+Installing from source on Unix-like platforms requires `cmake`, `git`, and a recent version of a C++ compiler (it was tested on `g++` version 10.5.0).
```

Comment:
Very nice.   Maybe reword to 

```suggestion
Installing from source on Unix-like platforms requires `cmake`, `git`, and a recent-enough C++ compiler (`g++` version 10.* or above).
```

We may want to add the `clang++` version too but I am unsure which it is. Maybe `clang++-16` ?